### PR TITLE
Added Simplified Chinese Translation

### PR DESCRIPTION
**Pull request by ahdg6**
Added Simplified Chinese Translation

### CODE REVIEW

File: 1.16.5/lang/presets/minecraft/zh_CN.conf [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    player-item-drop="控制 玩家 是否可以 丢弃 物品。"
+    player-item-pickup="控制 玩家 是否可以 拾取 物品。"
+    player-itemframe-interact="控制 玩家 是否可以与 物品展示框 交互。"
+    player-itemhanging-place="控制 玩家 是否可以放置诸如 物品展示框 等悬挂式物品。"
+    player-netherportal-use="控制 玩家 能否使用 地狱传送门。"
+    player-teleport-from="控制 玩家 能否通过 传送 离开 该领地。"
+    player-teleport-to="控制 玩家 能否通过 传送 进入 该领地。"
+    player-villager-damage="控制 玩家 能否对 村民 造成伤害。"
+    pvp="控制 PvP 对战是否允许。"
+	ravager-block-break="控制 掠夺者 在 突袭 过程中是否可以 破坏方块。"
```
Comment (kencinder) [inline 1.16.5/lang/presets/minecraft/zh_CN.conf]: Tab needs to be 4 spaces

File: 1.16.5/lang/presets/minecraft/zh_CN.conf [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    snow-fall=雪花飘落
+    snow-melt=雪融化
+    snowman-trail=雪人的路径轨迹
+    soil-dry=土壤变干
+    spawn-ambient=生成环境生物
+    spawn-animal=生成动物
+    spawn-aquatic=生成水生生物
+    tnt-block-explosion=tnt方块爆炸
+    tnt-entity-explosion=tnt实体爆炸
+	turtle-egg-hatch=海龟蛋孵化
```
Comment (kencinder) [inline 1.16.5/lang/presets/minecraft/zh_CN.conf]: Tab needs to be 4 spaces

**Comment by kencinder** [comment]
Looks good aside from those 2 lines needing fixed, I can't edit so you'll need to fix them before I can merge.

**Comment by ahdg6** [comment]
Fixed

**Pull request merged**